src/layers/EarthEngine.js

```diff
@@ -20,6 +20,7 @@ class EarthEngine extends Layer {
             this._map = map
 
             if (map.styleIsLoaded()) {
+                this._isLoading = true
```

Comment:
@HendrikThePendric Map.js no longer checks the loading status of Bing, EarthEngine or ServerCluster layers. If this check isn't needed anymore then we should remove the _isLoading flags

Reply:
@jenniferarnesen  I removed this check in https://github.com/dhis2/maps-gl/pull/552/commits/06fd528dafc5166f728c02f283c319bbc3cf827a because I kept seeing `._isLoading` being `undefined`. I now see I was simply not testing the right maps. So I'll push a commit that returns the check to `Map.js`